### ISSUE TITLE
Change steering council language about "ultimately make the final decisions."

### ISSUE THREAD

**Issue by fperez** [jupyter/jupyter.github.io#46]
When the site was in draft, I complained about this sounding very authoritarian and completely contrary to the sprit of our community, but I was told that this was just temporary placeholder text.  Instead, it went live.

Let's please _not_ have that language anywhere on the site: it sends _completely_ the wrong message to our community.

Ping @ellisonbg.

**Comment by minrk** [comment]
What's the right language for this? "Take responsibility when consensus cannot be reached"? cc @willingc as well.

**Comment by parente** [comment]
A link to the governance docs / repo would be good in that section too. People / companies need to understand that even council membership can be obtained.

**Comment by ellisonbg** [comment]
Min, I like that way of putting it, but maybe add "Take responsibility for
the decision..."

On Mon, Oct 12, 2015 at 10:17 AM, Peter Parente <notifications@github.com>
wrote:

> A link to the governance docs / repo would be good in that section too.
> People / companies need to understand that even council membership can be
> obtained.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyter.github.io/issues/46#issuecomment-147410980>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by willingc** [comment]
@cameronoelsen Typo in Steering Council description. Should be "led" in this sentence.
Our team is primarily lead by 10 steering committee members who ultimately make the final decisions.

**Comment by willingc** [comment]
How about this wording? 

Technical decisions are guided by open community discussion. 

The steering council (link to governance docs as suggested by @parente), as a representative of the greater community, may make decisions to benefit the project's overall health and strategic direction. BDFL Fernando Perez may take responsibility for making a decision when unusual, and likely infrequent, situations warrant, such as steering council deadlock or a time sensitive response.

**Comment by cameronoelsen** [comment]
I will fix this right when I get back in the office around 11 ☺️

**Comment by fperez** [comment]
I like where @willingc is going, thanks! That's precisely what I was looking for, we want to convey that the role of the SC/BDFL is:

- primarily one of sustaining and  nurturing a healthy and open community, where participation in discussion is encouraged from all, where new voices are welcome, and where a diversity of opinion and participants is explicitly desired.  We encourage respectful dissent, we value disagreement as a sign of creativity, and we try to steer that community towards better solutions to the many technical and governance problems the project deals with.

- only when necessary, do the SC/BDFL act as a *backstop* if the open community discussion and consensus process falters.  

Thank you so much, Carol, for your input.  Great perspective! Sorry to be going offline now, but I think this is going in a great direction, I trust you to find a good solution for now, we can always iterate again in a few weeks if fine-tuning is needed later.

I'm happy if you all close this with some version of the language in this discussion. If later I think it needs tuning, I'll take it upon myself to just open a PR.

Thanks everyone!

**Comment by minrk** [comment]
ping @cameronoelsen on this, since it's been a couple of weeks.

**Comment by cameronoelsen** [comment]
So do we want to add one or two buttons under the material cards (names of the members on the steering council) that leads outwards to documents describing how we are composed as an organization? As for the supporting subhead, how is this:
"Our team is primarily led by 10 steering committee members who ultimately, when open community discussion and consensus process falters, take responsibility for the final decision."

**Comment by willingc** [comment]
@cameronoelsen I think @fperez wanted a warmer, more collaborative tone for the Steering Council and Governance sections.

<img width="1152" alt="screen shot 2015-10-26 at 4 10 28 pm" src="https://cloud.githubusercontent.com/assets/2680980/10745118/2304ea84-7bfc-11e5-9d21-ac9199cd23a3.png">

Your suggested wording with minor edits would work for steering council:

"Our team is primarily led by 10 steering committee members who, when open community discussion falters on reaching consensus, take responsibility for the project direction and final decision."

<img width="1179" alt="screen shot 2015-10-26 at 4 08 06 pm" src="https://cloud.githubusercontent.com/assets/2680980/10745082/d8182ed2-7bfb-11e5-9991-58fc0d27cc83.png">

Under Governance -> Decisions, perhaps just use @fperez's suggested wording:

The Steering Council fosters project governance "... as primarily one of sustaining and nurturing a healthy and open community, where participation in discussion is encouraged from all, where new voices are welcome, and where a diversity of opinion and participants is explicitly desired. We encourage respectful dissent, we value disagreement as a sign of creativity, and we try to steer that community towards better solutions to the many technical and governance problems the project deals with."

Perhaps also followed by the existing wording too.

cc/ @minrk @ellisonbg @fperez

**Comment by ellisonbg** [comment]
pinging @fperez on this one.

When @cameronoelsen and I work on the website, I usually take passes on the actual content and its language. @cameronoelsen tries to fill in the content when he can, but I think he should focus on the visual design and work with us on the content side of things - especially for stuff like this.

@fperez you pinged me on this one at the start and I should have taken a stab and fixing the content myself but it dropped off my radar in my travels. Do you want to me to submit a PR with a potential fix or would you like to do that?
